cypress/integration/smoke/tx_history.spec.js

```diff
@@ -0,0 +1,234 @@
+const SAFE = 'rin:0x87a57cBf742CC1Fc702D0E9BF595b1E056693e2f'
+
+// Logo pathnames
+const CONTRACT_INTERACTION = '/app/static/media/custom.6f7b401c.svg'
```

Comment:
Isn't 6f7b401c changed from build to build?

Reply:
I've adjusted it to compare only the intial name of the file.